src/main/scala/mesosphere/marathon/raml/AppConversion.scala

```diff
@@ -80,12 +81,13 @@ trait AppConversion extends ConstraintConversion with EnvVarConversion with Heal
     )
   }
 
-  def resources(cpus: Option[Double], mem: Option[Double], disk: Option[Double], gpus: Option[Int]): Resources =
+  def resources(cpus: Option[Double], mem: Option[Double], disk: Option[Double], gpus: Option[Int], networkBandwidth: Option[Int]): Resources =
```

Comment:
Don't you want a Double here as well to express G/M/K/bps ?

Reply:
@pierrecdn , good remark but there won't be any unit parsing (at least at the beginning) to keep things simple. Plus, it would unlikely be supported in a generic implementation of custom resources anyway so I think it's gonna stay like this unless we decide "bytes per second" is not the default unit.

Reply:
We might want to choose "Kbps" as the default unit though. I'll wait feedbacks from my colleagues.

Reply:
After second thought, discard my argument regarding generic custom resources since we might want to implement the parsing at the level of the user request and leave the amount as "bytes per second" at the level of Mesos.